actions/open.go

```diff
@@ -0,0 +1,45 @@
+package actions
```

Comment:
@Dzalevski @FilipNikolovski This handler returns opens for certain campaign, isn't it better if it is in `campaign.go`?